docs/driver-builds.md

```diff
@@ -1,48 +1,42 @@
 ### Upstream drivers flow
+The jobs involved in building drivers and how they relate to each other looks
+a bit like this:
 
-<img src="./imgs/driver-flow.svg">
+<img src="./imgs/driver-build-overview.svg">
 
 <details>
 <summary>raw plantuml</summary>
 
 ```
 @startuml
 
-frame "GCP" {
-    component [driver cache] as dcache
-    component [driver bundles] as dbundles
-    component VM
-    component [support package cache] as supacache
+frame GHA {
+    component [Drivers Build] as dbuild
+    component [Build and push support-packages] as buildsupa
+    component [Push new drivers] as push
 
-    VM -.> supacache: Pushes created packages
-}
+    [collector-slim] --> [collector-full]
+    dbuild --> [collector-full]
 
-frame OSCI {
-    frame images {
-        component [Drivers Build] as dbuild
-
-        dbundles ..> dbuild
-        [collector-slim] -> [collector-full]
-        dbuild --> [collector-full]
-    }
-
-    frame tests {
-        component [integration tests] as its
-        component [push drivers] as dpush
-        component [update support package] as updatesupa
-
-        [collector-full] --> its
-        dbuild --> dpush
-        dpush ..> dcache
-        dcache ..> dbuild
-        dbuild --> updatesupa
-        updatesupa --> VM : Runs on
-    }
+    dbuild -> [Check compiled drivers]
+    dbuild --> push
+    push -> buildsupa
 }
+
 @enduml
+
 ```
 </details>
 
+In order to build the required drivers, the `Drivers build` step will pull
+kernel bundles from GCP. The bundles are tarballs containing the required
+headers for each kernel version supported, these are created by our
+[crawler](https://github.com/stackrox/kernel-packer).
+
+Once building the drivers is done, 3 follow up steps will be triggered:
```

Comment:
It's a bit weird that the previous step was "download bundles", and the following is "once building is done". Maybe for completeness rearrange sections to be more consistent?

Reply:
I've moved this paragraph to its own `Post-build jobs` section, hope this makes it a bit easier to follow.